Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -2700,6 +2703,42 @@ namespace Babylon
                 std::vector<xr::System::Session::Frame::InputSource::Identifier> squeezeStarts{};
                 std::vector<xr::System::Session::Frame::InputSource::Identifier> squeezeEnds{};
 
+                if (frame.EyeTrackerSpace.first)
```

Comment:
>first [](http://example.com/codeflow?start=42&length=5)

i really feel like this should be a struct field, it is really unclear what this first value is for